editor/editor_audio_buses.h

```diff
@@ -108,6 +108,7 @@ class EditorAudioBus : public PanelContainer {
 	void _effect_edited();
 	void _effect_add(int p_which);
 	void _effect_selected();
+	void _effects_deselected();
```

Comment:
It's a callback for focus lost, so it needs a more accurate name.

Reply:
I tried `effects->connect("empty_clicked", callable_mp(effects, &Tree:deselect_all)`, and got an error `Method expected 0 arguments, but called with 2`. I'm very rusty on cpp, so I don't know if there's a good way to make like... a closure or something to take in more arguments, so I'll just go ahead and rename 😅

Reply:
Did you specifically type that? If so you missed a colon: `effects->connect("empty_clicked", callable_mp(effects, &Tree::deselect_all));`

Reply:
I had the two colons in the actual code, I just retyped it out like an idiot.

But also my bad, `focus_exited` takes no arguments, I'll remove the extra function 🤦

Reply:
You can do `effects->connect("empty_clicked", callable_mp(effects, &Tree::deselect_all).unbind(2));` if you want to use this signal

Reply:
Ohh, got it thanks!